src/services/pipewire/node.cpp

```diff
@@ -116,21 +129,29 @@ void PwNode::unbindHooks() {
 void PwNode::initProps(const spa_dict* props) {
 	if (const auto* mediaClass = spa_dict_lookup(props, SPA_KEY_MEDIA_CLASS)) {
 		if (strcmp(mediaClass, "Audio/Sink") == 0) {
-			this->type = PwNodeType::Audio;
+			this->type = PwNodeType::AudioSink;
 			this->isSink = true;
 			this->isStream = false;
 		} else if (strcmp(mediaClass, "Audio/Source") == 0) {
-			this->type = PwNodeType::Audio;
+			this->type = PwNodeType::AudioSource;
 			this->isSink = false;
 			this->isStream = false;
 		} else if (strcmp(mediaClass, "Stream/Output/Audio") == 0) {
-			this->type = PwNodeType::Audio;
+			this->type = PwNodeType::AudioOutStream;
 			this->isSink = false;
 			this->isStream = true;
 		} else if (strcmp(mediaClass, "Stream/Input/Audio") == 0) {
-			this->type = PwNodeType::Audio;
+			this->type = PwNodeType::AudioInStream;
 			this->isSink = true;
 			this->isStream = true;
+		} else if (strcmp(mediaClass, "Video/Sink") == 0) {
+			this->type = PwNodeType::VideoSink;
+			this->isSink = false;
+			this->isStream = false;
+		} else if (strcmp(mediaClass, "Video/Source") == 0) {
+			this->type = PwNodeType::VideoSource;
+			this->isSink = false;
+			this->isStream = false;
```

Comment:
I omitted `Audio/Duplex` as I was unsure what combination of isSink and isStream it should be

Reply:
So should I be renaming this enum to `PwNodeMediaClass`/`PwNodeClass` and add helper functions `isAudio`, `isVideo` or have PwNodeClass and PwNodeType simultaneously ?

I found using the boolean combination isStream and isSink to be a bit counter intuitive, so I'd greatly appreciate something that'd report the pipewire class more closely

Reply:
I'd be ok with a bitflag set replacing the booleans as long as we still expose the booleans through qml.

e.g. `Audio | Video | Stream | Source | Sink`.

Reply:
Will update the Pr with that then, but does it make more sense over an enum considering many of the combinations are invalid?

Reply:
Flags are useful for testing imo, but you can have both.
e.g. `AudioInStream` = `Audio | Sink | Stream`